client/src/services/bpmnService.ts

```
@@ -1,651 +1,651 @@
+      if (node.type === 'lane' || node.type === 'pool') return;
+      
+      const nodeId = node.id;
+      const label = node.data.label || nodeId;
+      
+      // Get outgoing edges
+      const outgoing = edges.filter(e => e.source === nodeId);
+      const incoming = edges.filter(e => e.target === nodeId);
+      
+      switch (node.type) {
+        case 'event':
+          const eventType = node.data.eventType || 'start';
```

Biome (2.4.16)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)